### PR TITLE
Welcome page

### PR DESCRIPTION
**Pull request by kihlaj**
**What is done:**

- [X] Feature added
- [ ] Bug fixed
- [ ] Code refactored
- [ ] Documentation updated
- [ ] Tests added
- [ ] Other (please specify):

### PR Purpose

### CODE REVIEW

**Comment by DogukanGun** [comment]
Could u solve the conflicts?

**Comment by kihlaj** [comment]
Should I merge with main?

**Comment by DogukanGun** [comment]
> Should I merge with main?

Yes, you should merge with main and then solve your conflicts in your local environment. After that please push your code again.

File: .github/auto_assign.yml [hunk 1]
```diff
@@ -5,9 +5,9 @@ addReviewers: true
 addAssignees: true
 
 # A list of reviewers to be added to pull requests (GitHub user name)
-reviewers: 
+reviewers:
```
Comment (DogukanGun) [inline .github/auto_assign.yml:8]: Please revert the changes in .github file

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -7,4 +7,4 @@
 - [ ] Tests added
 - [ ] Other (please specify):
 
-### PR Purpose
\ No newline at end of file
+### PR Purpose
```
Comment (DogukanGun) [inline .github/pull_request_template.md:10]: Please revert the changes in .github file

File: .github/workflows/auto-assign.yml [hunk 3]
```diff
@@ -11,9 +11,9 @@ jobs:
       issues: write
       pull-requests: write
     steps:
-    - name: 'Auto-assign issue'
-      uses: pozil/auto-assign-issue@v1
-      with:
+      - name: 'Auto-assign issue'
```
Comment (DogukanGun) [inline .github/workflows/auto-assign.yml:14]: Please revert the changes in .github file

File: .github/workflows/eslint.yml [hunk 4]
```diff
@@ -11,10 +11,10 @@ name: ESLint
 
 on:
   push:
-    branches: [ "main" ]
+    branches: ['main']
```
Comment (DogukanGun) [inline .github/workflows/eslint.yml:14]: Please revert the changes in .github file

File: .github/workflows/eslint.yml [hunk 5]
```diff
@@ -11,10 +11,10 @@ name: ESLint
...
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ "main" ]
+    branches: ['main']
```
Comment (DogukanGun) [inline .github/workflows/eslint.yml:17]: Please revert the changes in .github file

File: .github/workflows/pull-request.yml [hunk 6]
```diff
@@ -5,4 +5,4 @@ jobs:
   add-reviews:
     runs-on: ubuntu-latest
     steps:
```
Comment (DogukanGun) [inline .github/workflows/pull-request.yml:7]: Please revert the changes in .github file

File: app/globals.css [hunk 7]
```diff
@@ -1,27 +1,16 @@
...
-  background: linear-gradient(
-      to bottom,
-      transparent,
-      rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
+@tailwind components;
+@tailwind utilities;
+
+@layer utilities {
```
Comment (DogukanGun) [inline app/globals.css:12]: This might be done in tailwind classes, could u check ?

File: components/CustomTextfield/Contact.tsx [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+function Contact(): React.JSX.Element {
+  return (
+    <div>
+      <form className="flex h-[624.04px] w-[514.56px] flex-col rounded-[10px] bg-white p-12">
+        <span className="text-sm font-bold uppercase leading-relaxed tracking-widest text-violet-950">
+          Your Name
+        </span>
+        <input
+          className="borderBlack mt-4 h-14 rounded-full border-2 border-gray-200 px-4"
+          name="name"
```
Comment (DogukanGun) [inline components/CustomTextfield/Contact.tsx:14]: Could u make this a component?

File: components/CustomTextfield/Contact.tsx [hunk 9]
```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import CustomButton from '@/components/CustomButton'
+
+function Contact(): React.JSX.Element {
+  return (
+    <div>
+      <form className="flex h-[624.04px] w-[514.56px] flex-col rounded-[10px] bg-white p-12">
```
Comment (DogukanGun) [inline components/CustomTextfield/Contact.tsx:8]: Could u change `h-[624.04px] w-[514.56px] rounded-[10px]` ?  We should use values in tailwind predefined classes.

File: components/CustomFooter/index.tsx [hunk 10]
```diff
@@ -0,0 +1,97 @@
...
+          </div>
+          <div className="mb-10 mt-48 border bg-white bg-opacity-20" />
+          <div className="mb-20 flex justify-between">
+            <div className="text-sm leading-7 text-emerald-50">
+              © 2023 NexArb. All Rights Reserved.
+            </div>
+            <div className="flex flex-row gap-6">
+              {FooterSocials.map((e) => (
+                <Link href="/" key={e.svg}>
+                  <div className="flexCenter h-[35.70px] w-[35.70px] rounded-lg bg-white bg-opacity-20">
```
Comment (DogukanGun) [inline components/CustomFooter/index.tsx:84]: Could u change `h-[35.70px] w-[35.70px]` ? We should use values in tailwind predefined classes.

File: components/CustomFooter/index.tsx [hunk 11]
```diff
@@ -0,0 +1,97 @@
...
+                </div>
+              </div>
+              <div className="mt-14 flex flex-row gap-5">
+                <Image
+                  src={'/img/fa-building-o.svg'}
+                  alt="logo"
+                  width={19}
+                  height={19}
+                />
+                <div className="h-[85.39px] w-[275.89px] text-base font-normal leading-relaxed">
```
Comment (DogukanGun) [inline components/CustomFooter/index.tsx:68]: Could u change `h-[85.39px] w-[275.89px]` ? We should use values in tailwind predefined classes.

File: components/CustomFooter/index.tsx [hunk 12]
```diff
@@ -0,0 +1,97 @@
...
+        quality={95}
+      />
+      <div className="flexCenter z-20 flex-col ">
+        <div className="mt-40 flex flex-col">
+          <div className="mx-auto flex w-full max-w-[1440px] flex-1 gap-28 px-6 lg:px-20 xl:px-0">
+            <div>
+              <div className="mt-14 text-[42px] font-bold leading-[52px]">
+                Let’s Keep in Touch
+              </div>
+              <div className="mt-8 h-[140.14px] w-[436.83px] text-[22px] font-medium leading-loose">
```
Comment (DogukanGun) [inline components/CustomFooter/index.tsx:34]: Please tailwind predefined classes.

File: components/CustomFooter/index.tsx [hunk 13]
```diff
@@ -0,0 +1,97 @@
...
+                quickly and easily.
+              </div>
+              <div className="mt-20 flex flex-row gap-5">
+                <Image
+                  src={'/img/fa-phone.svg'}
+                  alt="logo"
+                  width={19}
+                  height={19}
+                />
+                <div className="h-[28.46px] w-[263.85px] text-base font-normal leading-relaxed">
```
Comment (DogukanGun) [inline components/CustomFooter/index.tsx:46]: Please tailwind predefined classes.

File: components/CustomFooter/index.tsx [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+        alt="mountains"
+        fill
+        className="md:hidden"
+        quality={95}
+      />
+      <div className="flexCenter z-20 flex-col ">
+        <div className="mt-40 flex flex-col">
+          <div className="mx-auto flex w-full max-w-[1440px] flex-1 gap-28 px-6 lg:px-20 xl:px-0">
+            <div>
+              <div className="mt-14 text-[42px] font-bold leading-[52px]">
```
Comment (DogukanGun) [inline components/CustomFooter/index.tsx:31]: Please tailwind predefined classes.

**Review by DogukanGun — CHANGES_REQUESTED** [review]

**Review by DogukanGun — APPROVED** [review]

**Pull request merged**